README.md

```diff
@@ -2,20 +2,32 @@
 
 ## EYE1
 
-The legal possession of this `CRED` token is [recordat]
+The legal possession of this `CRED` token is [recordat]() of the _social engagement_, skills, and side-projects of the owner. In conjunction with srCRED [^1] ¿TBA?, it is LexDAO's assessment (passive and active/explicit voting) of participating in legal engineering activities including:
+- POAP attendance in studygroups or hackerthons;
+- TCR - indicating preference in token curated registries [^2] for an activity or priority;
+- funded activities such as working for membership or gaining a levelup via hackathon quest.
+
+## HOW2
+
 An outline of the current workflow to deploy a simple custom ERC 721 NFT Token
 
-# Plan Framework and Concept
------------------
-## Date Marker
-The plan frameowork and concept were last updated 2023-03-08
+Membership Flow Mockup -  [Excalidraw link](https://excalidraw.com/#room=2129b2214566ba2e246d,ykrk2lDXqfz4XnEUXArLBQ)
+  ![Membership Join Flow](https://user-images.githubusercontent.com/106759485/229314079-b935c399-103d-48dc-9dc1-b703409a6262.png)
 
+> [!CAUTION]
+> There will be progressive upgrade in the associated contracts ... currently by minting a new token every year, the information loses continuity so an important worktask is to find the right improvements in ERC721 such as handles and not pointers.
```

Comment:
We don't need to update the 721.  I want to use it as a key for the 1155.  Which is where all (most) upgrades take place.  Think of the 721 as your keycard and the 1155(s) as your network tokens.  Like a certificate authority on a network.  Then you can bolt 4337 and 6551 to the 1155.  All are compatible together to make a proper system architecture.